lago.spec.in

```diff
@@ -50,6 +50,12 @@ install -p -D -m 644 etc/firewalld/services/* %{buildroot}%{_sysconfdir}/firewal
 %package -n python-%{name}
 Summary: Library to perform lago operations
 BuildArch: noarch
+
+%if 0%{?fedora} == 24
+Requires: python2-crypto = 2.6.1-10.fc24
```

Comment:
I would separate the ``Requires`` and ``BuildRequires`` to different sections(like the rest of the file), and add a link to the BZ.